docs/electron.rst

```diff
@@ -156,6 +156,18 @@ is not required.
 ``org.electronjs.Electron2.BaseApp`` since ``branch/23.08`` comes with
 ``libnotify>=0.8.0``
 
+.. _use-correct-desktop-filename:
+
+Using correct desktop file name
+-------------------------------
+It's important for Linux Applications to know the correct name of their desktop file. If not, it can led to problems e.g. a missing Icon under Wayland.
```

Comment:
```suggestion
It's important for Linux applications to know the correct name of their desktop file. If not, it can led to problems e.g. a missing icon under Wayland.
```